src-electron/generator/helper-zcl.js

```diff
@@ -323,6 +323,17 @@ async function zcl_struct_items_by_struct_and_cluster_name(
   options
 ) {
   let packageIds = await templateUtil.ensureZclPackageIds(this)
```

Comment:
should we update the documentation to explain how global clusters take effect? then cluster name has no meaning.

Reply:
Done.